src/main/java/io/reactivex/internal/schedulers/SchedulerRunnableWrapper.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2016-present, RxJava Contributors.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See
+ * the License for the specific language governing permissions and limitations under the License.
+ */
+
+package io.reactivex.internal.schedulers;
+
+public interface SchedulerRunnableWrapper extends Runnable {
+
+    Runnable getWrappedRunnable();
```

Comment:
Please add a short description of this method and the `@Nullable` annotation (some terminated wrappers do null out their wrapped `Runnable` instance.

Reply:
ok